packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -492,7 +497,7 @@ const waitForVisible = async (
 	}
 
 	const element = await driver.elementsByXPath( elementLocator );
-	if ( element.length !== 1 ) {
+	if ( element.length === 0 ) {
```

Comment:
I think it would be interesting adding a parameter to the function to configure whether the function returns the first element or the elements array. It could be useful for those cases where we need to pick one of the elements, like for example in the [`getLastBlockVisible` helper function](https://github.com/WordPress/gutenberg/blob/36adacdf183ed04c05eed89b28711a7b11655666/packages/react-native-editor/__device-tests__/pages/editor-page.js#L151-L156).